classes/order/OrderInvoice.php

```diff
@@ -112,7 +112,7 @@ class OrderInvoiceCore extends ObjectModel
             'total_wrapping_tax_excl' => ['type' => self::TYPE_FLOAT],
             'total_wrapping_tax_incl' => ['type' => self::TYPE_FLOAT],
             'shop_address' => ['type' => self::TYPE_HTML, 'validate' => 'isCleanHtml', 'size' => 1000],
-            'note' => ['type' => self::TYPE_STRING, 'validate' => 'isCleanHtml', 'size' => 65000],
+            'note' => ['type' => self::TYPE_HTML],
```

Comment:
If I am not wrong, this change has huge impacts? Modify a column of objectmodel OrderInvoice?

From what I see in class ObjectModel, it will modify how `formatValue()`,  `getHtmlFields()`

Also if someone is analyzing the fields of an `$invoice` objectmodel and he is using
```
        if (array_key_exists('fields', $this->def)) {
            foreach ($this->def['fields'] as $name => $field) {
                if (is_array($field) && array_key_exists('type', $field) && self::TYPE_STRING === $field['type']) {
                    // do something
                }
            }
        }
```
it will not work anymore

So it looks like a BC break to me 🤔  .

Also if we remove `'validate' => 'isCleanHtml', 'size' => 65000` what are the default values for this field now?

Reply:
It's not a BC Break as we don't define that field type of objectmodel are parts of our API.

Reply:
I'm not sure it's a BC either but I would keep `'size' => 65000` though 🤔